cmd/limactl/shell.go

```diff
@@ -108,7 +109,16 @@ func shellAction(cmd *cobra.Command, args []string) error {
 	}
 	logrus.Debugf("changeDirCmd=%q", changeDirCmd)
 
-	script := fmt.Sprintf("%s ; exec $SHELL --login", changeDirCmd)
+	shell, err := cmd.Flags().GetString("shell")
+	if err != nil {
+		return err
+	}
+	if shell == "" {
+		shell = "$SHELL"
```

Comment:
Can we quote this too?

Reply:
> Can we quote this too?

Sure, but here we need `"$SHELL"` quotes and not `shellescape.Quote` because in this case we do want variable interpolation.